Build system regenerates setup.py and paver-minilib.zip.
This modify original sources.
We don't need this since they are in the release tarball.
--- a/pavement.py
+++ b/pavement.py
@@ -150,8 +150,8 @@
 
 @task
 @needs("eyed3_info",
-       "generate_setup",
-       "minilib",
+       #"generate_setup",
+       #"minilib",
        "setuptools.command.build")
 def build():
     '''Build the code'''
